src/redux/books/books.js

```diff
@@ -1,50 +1,75 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
```

Comment:
While you did well working on some of the project requirements I notice you disabled eslint. It is against microverse policy to disable linters for linters help us write clean code. so it would be nice if you enabled eslint in this file.